browser/modules/AboutNewTab.jsm

```diff
@@ -65,8 +68,12 @@ let AboutNewTab = {
    *          history tiles feature.
    */
   customize: function(message) {
-    NewTabUtils.allPages.enabled = message.data.enabled;
-    NewTabUtils.allPages.enhanced = message.data.enhanced;
+    if (message.data.enabled !== undefined) {
```

Comment:
We just want a "truthy" check here, right (i.e., `if (message.data.enabled)`)? Or is there some reason to explicitly check for undefined?